app/templates/components/ui-table.hbs

```diff
@@ -15,7 +17,14 @@
   <div class="ui row">
     <div class="{{themeInstance.classes.innerTableWrapper}}">
       <table class="{{themeInstance.classes.table}}">
-        <thead class="{{if noHeaderFilteringAndSorting 'table-header-no-filtering-and-sorting'}} {{themeInstance.classes.thead}}">
+        <thead
+          class="{{if
+              noHeaderFilteringAndSorting
+              "table-header-no-filtering-and-sorting"
+            }}
+
+            {{themeInstance.classes.thead}}"
+        >
```

Comment:
Same question here, what was wrong in the initial format as long as it was not encountering any template linting errors?
What was the motivation behind changing it?

Reply:
@uds5501 I guess my text editor formatted that automatically. Will correct it.